### PR TITLE
removing px from core values

### PR DESCRIPTION
**Pull request by brianrclay**
@phantomxc

### CODE REVIEW

File: src/tokens/core.js [hunk 1]
```diff
@@ -37,57 +37,57 @@ export default {
     TransparentScrimDarker: "rgba(0, 0, 0, 0.50)",
   },
   BoxShadow: {
-    Low: ' 0px 1px 3px rgba(87, 102, 117, 0.2)',
-    Medium: ' 0px 3px 8px rgba(87, 102, 117, 0.15)',
-    High: ' 0px 10px 20px rgba(87, 102, 117, 0.16)',
-    Top: ' 0px -1px 3px rgba(87, 102, 117, 0.2)',
-    Left: ' -1px 0px 3px rgba(87, 102, 117, 0.2)',
-    Right: ' 1px 0px 3px rgba(87, 102, 117, 0.2)',
-    Focus: '0 0 0 2px rgba(82, 138, 224, 0.8)',
```
Comment (phantomxc) [inline src/tokens/core.js:46]: It doesn't look like we'll be able to use these definitions in react native so these can be put back.

File: src/tokens/core.js [hunk 2]
```diff
@@ -37,57 +37,57 @@ export default {
...
+    Top: ' 0 -1 3 rgba(87, 102, 117, 0.2)',
+    Left: ' -1 0 3 rgba(87, 102, 117, 0.2)',
+    Right: ' 1 0 3 rgba(87, 102, 117, 0.2)',
+    Focus: '0 0 0 2 rgba(82, 138, 224, 0.8)',
   },
   BorderRadius: {
-    Small: '2px',
-    Medium: '4px',
-    Large: '16px',
+    Small: '2',
```
Comment (phantomxc) [inline src/tokens/core.js]: These can all be updated to exclude the quotations.

**Review by inphomercial — APPROVED** [review]
After some offline chat with @phantomxc, this looks like it's gonna be a good solution to allow for both our web components and our react-native components to share the same design tokens.

**Comment by phantomxc** [comment]
If you want to bump the version number to 3.0.0 in the package.json file, I'll approve and publish.

**Comment by phantomxc** [comment]
:shipit:

**Pull request merged**